.github/workflows/distcheck.yml

```
@@ -0,0 +1,35 @@
+name: Autotools distcheck
+
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+
+jobs:
+  distcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Linux dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y libtool-bin libogg-dev doxygen libxml2-utils w3c-sgml-lib
+
+      - name: Build with Autotools
+        run: |
+          ./autogen.sh
+          ./configure
```

```suggestion
          ./configure --disable-thorough-tests
```
The test suite is already run in all it's facets, so I think running a slimmed down one here should be fine. After all, we mostly want to know whether it builds and runs, I guess?

---

I don't think this will have any effect. `make distcheck` invokes configure again; the outer build options aren't propagated.

There is an `(AM_)DISTCHECK_CONFIGURE_FLAGS` you can set if you want to do this.

---

Ok, probably best to leave it like this then